hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/security/DelegationTokenRenewer.java

```diff
@@ -627,6 +634,7 @@ private boolean skipTokenRenewal(Token<?> token)
     if (identifier == null) {
       return false;
     }
+    expiredTime.set(identifier.getMaxDate());
```

Comment:
Do we just to reset the expireTime only when the result of `skipTokenRenewal` is false?